src/List View/code.rs

```diff
@@ -0,0 +1,81 @@
+use crate::workbench;
+use gtk;
+use gtk::glib;
+use gtk::prelude::*;
+use std::cell::Cell;
+
+pub fn main() {
+    // Required by gtk::StringList::new(), otherwise crashes
+    gtk::init().unwrap();
+
+    let list_view: gtk::ListView = workbench::builder().object("list_view").unwrap();
+    let add: gtk::Button = workbench::builder().object("add").unwrap();
+    let remove: gtk::Button = workbench::builder().object("remove").unwrap();
+
+    // https://doc.rust-lang.org/std/cell/struct.Cell.html
+    // "A mutable memory location."
+    // Makes possible accessing and changing values from signal handlers
+    let item = Cell::new(1);
+
+    // Model
+    let strings_model =
+        gtk::StringList::new(&["Default Item 1", "Default Item 2", "Default Item 3"]);
+    let model = gtk::SingleSelection::new(Some(strings_model));
+
+    //View
+    model.connect_items_changed(move |_model, position, removed, added| {
+        println!(
+            "position: {}, Item removed? {} Item added? {}",
```

Comment:
```suggestion
            "position: {}, Item removed? {}, Item added? {}",
```

Reply:
@Hofer-Julian This one produces quite interesting values, it differs from other languages. Is that expected based on your gtk-rs knowledge?